spec/services/course_search_service_spec.rb

```diff
@@ -424,4 +434,64 @@
       end
     end
   end
+
+  describe "10 miles diff" do
+    context "university course vs non university course" do
+      null_island = { latitude: 0, longitude: 0 }
+
+      let(:university_course) do
+        create(:course, provider: university_provider,
+          site_statuses: [build(:site_status, :findable, site: site)],
+          enrichments: [build(:course_enrichment, :published)])
+      end
+
+      let(:non_university_course) do
+        create(:course, provider: non_university_provider,
+          site_statuses: [build(:site_status, :findable, site: site2)],
+          enrichments: [build(:course_enrichment, :published)])
+      end
+
+      let(:courses) do
+        [university_course, non_university_course]
+      end
+
+      let(:site) do
+        build(:site, **null_island)
+      end
+
+      let(:site2) do
+        build(:site, **null_island)
+      end
+
+      let(:university_provider) do
+        build(:provider, provider_type: :university, sites: [site])
+      end
+
+      let(:non_university_provider) do
+        build(:provider, provider_type: :scitt, sites: [site2])
+      end
+
+      before do
+        courses
+      end
+
+      let(:scope) do
+        Course.all
+      end
+
+      subject do
+        described_class.call(filter: null_island, sort: "distance", course_scope: scope)
+      end
+
+      it "returns correctly" do
+        expect(subject.count(:id)).to eq(2)
+        expect(subject.first.boosted_distance).to eq(-10)
```

Comment:
Do we want to be allowing negative values for this field. Shouldn't it return 0 if the calculated value is negative?

Reply:
If you have many that are within 10 miles, if we make them all 0 won't we lose our ranking ability?

Reply:
How I see it 
1. the `boosted_distance` is for the `ranking`
2. the `distance` is the actual `calculated` geometry
3. a `perceived distance` is needed for the perception/display to user

ie
`boosted_distance` is a negative value, `perceived distance` is therefore `0`
`boosted_distance` is a positive value, `perceived distance` is therefore `boosted_distance`